MAUI/DataForm/Working-with-DataForm.md

```diff
@@ -0,0 +1,575 @@
+---
+layout: post
+title: Working with DataForm in Maui DataForm control | Syncfusion
+description: Learn about the working with DataForm in Syncfusion .NET MAUI DataForm(SfDataForm) control in mobile and desktop applications from a single shared codebase.
+platform: Maui
+control: SfDataForm
+documentation: UG
+---
+
+# Working with DataForm in Maui DataForm (SfDataForm)
+
+## Auto-generating DataFormItems for the data field
+
+By default, the `SfDataForm.Items` will be generated based on the property type. For example, the `DataFormTextItem` will be created for the `string` type property. 
+
+The `DataFormItem` generation depends on the type and attribute defined for the property.
+The following tables lists the several types of `DataFormItem` and its constraints for auto generation:
+
+<table>
+<tr>
+<th>Generated DataFormItem Type</th>
+<th>Editor</th>
+<th>Data Type / Attribute</th>
+</tr>
+<tr>
+<td>
+DataFormTextItem
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Default DataFormItem generated for the String type and the properties with [DataType(DataType.Text)] attributes. 
+</td>
+</tr>
+<tr>
+<td>
+`DataFormMultilineTextItem`
+</td>
+<td>
+{{'[Editor](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/editor?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the String type and the properties with  [DataType(DataType.MultilineText)] attributes. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormPasswordItem
+</td>
+<td>
+{{'[Entry](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/entry?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for the String type and the properties with [DataType(DataType.Password)] attributes. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormDateItem
+</td>
+<td>
+{{'[DatePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/datepicker?view=net-maui-7.0)'| markdownify }}
+</td>
+<td>
+Generated for DateTime, DateTimeOffset, DateOnly type and properties with [DataType(DataType.Date)] and [DataType(DataType.DateTime)] attributes. 
+</td>
+</tr>
+<tr>
+<td>
+DataFormTimeItem
+</td>
+<td>
+{{'[TimePicker](https://learn.microsoft.com/en-us/dotnet/maui/user-interface/controls/timepicker?view=net-maui-7.0&tabs=windows)'| markdownify }}
+</td>
+<td>
+Generated for TimeSpan, TimeOnly, DateTime and DateTimeOffset property with [DataType(DataType.Time)] attribute. 
```

Comment:
Generated for TimeSpan or TimeOnly property type and properties with [DataType(DataType.Time)] attribute.

Reply:
content updated